### PR TITLE
Add the -f option and change way to create output

### PR DESCRIPTION
**Pull request by soimkim**
## Description
- Add the -f option to input the output file format.
- Change it to use FL Util's functions when generating output.
- Input Mode
    - **AS-IS**: If user just type enter, it asks user to re-enter (try 2 times).
                 Source and dependency path to be analyzed are inputted respectively.
    - **TO-BE**: If user just type enter in the input mode, it is assumed that nothing has been inputted.
                  Source and dependency path to be analyzed is input at once.

## Type of change
Please insert 'x' one of the type of change.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/fosslight_scanner/fosslight_scanner.py [hunk 1]
```diff
@@ -83,44 +87,44 @@ def set_sub_parameter(default_params, params):  # For dependency
...
-            success, error_msg = merge_excels(_output_dir, ouput_file)
+                dep_result_list = run_analysis(dep_path, set_sub_parameter(["DEP",
+                                                                            "-p", os.path.abspath(dep_path),
+                                                                            "-o", os.path.join(_output_dir, output_files["DEP"])],
+                                               dep_arguments), dep_main, "Dependency Analysis",
+                                               _output_dir, _executed_path)
+                sheet_list['SRC_FL_Dependency'] = dep_result_list.get('SRC')
+
+            output_file_without_ext = os.path.join(final_excel_dir, output_file)
+            success, msg = write_output_file(output_file_without_ext, OUTPUT_FILE_EXTENSION, sheet_list, scanner='FL-Source')
```
Comment (soimkim) [inline src/fosslight_scanner/fosslight_scanner.py:122]: @dd-jy , There is no scanner for FL-Scanner yet, so I set the scanner as `FL-Source`.

Comment (dd-jy) [inline src/fosslight_scanner/fosslight_scanner.py:122]: @soimkim 
It always generated xlsx file. please modify it (OUTPUT_FILE_EXTENSION).

Comment (dd-jy) [reply src/fosslight_scanner/fosslight_scanner.py:122]: FL dependency wants to change the sheet name from 'SRC' to 'SRC_FL_Dependency'.
Then the scanner parameter of write_output_file can be removed.

Comment (soimkim) [reply src/fosslight_scanner/fosslight_scanner.py]: It's my mistake. 😓 I fixed it.

**Pull request merged**